web/app/themes/mitlib-child/css/build

```diff
@@ -0,0 +1 @@
+../../../uploads/wp-scss/mitlib-child-build/
```

Comment:
The equivalent build and cache directories don't seem to be committed in the parent theme. I don't think it's particularly good/bad either way but I thought I'd check to see if you felt parent should change or these should be git ignored or if there is a reason the two themes should have this slight (but probably meaningless) difference?

Reply:
Hmm... this is an area where I'm not 100% confident in our setup, to be honest. That path (the uploads directory and everything under it) is gitignored, at least in my local, but I see the following:

```
% ls -l mitlib-child/css
build -> ../../../uploads/wp-scss/mitlib-child-build/
cache -> ../../../uploads/wp-scss/mitlib-child-cache/

% ls -l mitlib-parent/css
build -> ../../../uploads/wp-scss/mitlib-parent-build
cache -> ../../../uploads/wp-scss/mitlib-parent-cache
```

Within the `uploads/wp-scss` folder, there are eight folders - a build and cache folder for each of the four themes we had originally. The MOH theme has obviously been retired, so I can remove them, and the news theme doesn't use SCSS (so it can probably be removed too).

What do you have / would it be better to create an empty directory and a `.gitkeep` file to ensure these are created?